static/sass/_patterns_maas_modal.scss

```diff
@@ -104,7 +104,13 @@ $color-modal-controls: rgba($color-x-dark, .2);
 
   .vbox-next,
   .vbox-prev {
-    background-color: $color-x-light;
+    background: {
```

Comment:
This file was a just straight copy from maas, so didn't want to touch it.
But if it helps your sanity... ;)

Reply:
`./run test` all those lines.....